components/scream/src/share/tests/kokkos_utils_tests.cpp

```diff
@@ -66,6 +68,7 @@ TEST_CASE("team_utils", "[kokkos_utils]") {
 
 #ifdef KOKKOS_ENABLE_OPENMP
   const int n = omp_get_max_threads();
+  REQUIRE(n <= 16); // test will not work with more than 16 threads
```

Comment:
Why not? If the test is designed for no more than 16 and that makes sense, then it should do something other than make the unit test fail, so that we can still run other unit tests with a higher thread count.

Reply:
Yes, the test is designed for a maximum of 16 threads. That's a good point though, I should just skip the test if thrds>16, not fail.